alpha/apps/kaltura/lib/kContextDataHelper.php

```diff
@@ -268,9 +269,9 @@ private function setContextDataFlavorAssets($flavorTags)
 		if (is_null($this->asset))
 		{
 			if($flavorParamsIds && count($flavorParamsIds))
-				$flavorAssets = assetPeer::retrieveReadyByEntryIdAndFlavorParams($mediaEntryId, $flavorParamsIds, $flavorParamsNotIn);
```

Comment:
Based on what I see in the code, I am not sure this change is correct in the case of playlist